packages/charts/package.json

```diff
@@ -6,6 +6,7 @@
   },
   "description": "vaadin-charts",
   "license": "https://raw.githubusercontent.com/vaadin/web-components/master/packages/charts/LICENSE",
+  "cvdlName": "vaadin-chart",
```

Comment:
Should this be plural (as in the package name) ot not (web component tag name)?

Reply:
It has to match the web component's tag name for historical reasons. However, that only applies to the existing components and I guess it won't be an obligation for future components anymore as soon as the new license checker is no longer behind a feature flag.